llvm/lib/Target/RISCV/MCTargetDesc/RISCVMCCodeEmitter.cpp

```diff
@@ -119,7 +123,7 @@ void RISCVMCCodeEmitter::expandFunctionCall(const MCInst &MI,
   MCInst TmpInst;
   MCOperand Func;
   MCRegister Ra;
-  bool IsCap;
+  bool IsCap = false;
```

Comment:
This seems unrelated, should probably drop all the IsCap = false assignments below?

Reply:
removed :+1: